app/src/main/kotlin/co/netguru/baby/monitor/client/data/DataRepository.kt

```diff
@@ -1,66 +1,64 @@
 package co.netguru.baby.monitor.client.data
 
+import android.arch.lifecycle.LiveData
 import co.netguru.baby.monitor.client.application.scope.AppScope
 import co.netguru.baby.monitor.client.data.client.ChildDataEntity
 import co.netguru.baby.monitor.client.data.client.home.log.LogDataEntity
 import co.netguru.baby.monitor.client.data.communication.ClientEntity
 import co.netguru.baby.monitor.client.data.splash.AppState
 import co.netguru.baby.monitor.client.data.splash.AppStateHandler
 import io.reactivex.Completable
-import io.reactivex.Flowable
+import io.reactivex.Maybe
 import io.reactivex.Single
 import javax.inject.Inject
 
 @AppScope
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/netguru/baby-monitor-client-android/blob/636d317b685d72b06abf1b59ef55421cd7244cef/app/src/main/kotlin/co/netguru/baby/monitor/client/data/DataRepository.kt#L15"></span>Class ‘DataRepository’ with ‘12’ functions detected. Defined threshold inside classes is set to ‘11’</td>
    </tr>
  </tbody>
</table>